spec/models/comment_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  @user1 = User.new(name: 'Tom', photo: 'url here', bio: 'bio text here')
+  @user2 = User.new(name: 'Lilly', photo: 'url here', bio: 'bio text here')
+  @post1 = Post.new(author: @user1, title: 'Hello', text: 'this is my first post')
+
+  subject { Comment.create(post: @post1, author: @user2, text: 'Hi Tom!') }
```

Comment:
- [ ]  As requested by the previous reviewer, As required for the project to test any custom methods in the models, kindly ensure that you test the `#update_comments_counter` in the comment model :pray:

Reply:
The suggested test gave me a lot of tough time but I have learned a lot about Rails app testing while doing that.
Thank you